bitnami/kafka/templates/statefulset.yaml

```diff
@@ -472,6 +472,9 @@ spec:
           command:
             - java
           args:
+            {{- if .Values.metrics.jmx.extraJavaArgs }}
```

Comment:
LGTM but did you consider the same approach we apply for main containers where users are able to change whole `args` or `command`  values? Some lines above we are taking into account the diagnosticMode also:
```yaml
          ....
          {{- if .Values.%%MAIN_OBJECT_BLOCK%%.args }}
          args: {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.args "context" $) | nindent 12 }}
           {{- else }}
          ....
```